protoc-gen-grpc-gateway/gengateway/template.go

```diff
@@ -23,6 +23,14 @@ type binding struct {
 	*descriptor.Binding
 }
 
+// GetBinding returns the binding body
+func (b binding) GetBinding() string {
```

Comment:
I think naming this something more specific would be better off: `GetBodyFieldPath` or something. Since `Binding` is already a proper noun in this context, it'd be confusing to have a function named that which doesn't return the thing that it's named after. Make sense?

Normally I wouldn't be picky about the naming of this function, but since this isn't our codebase let's be good guests :)